tests/Feature/TagTest.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tests\Feature;
+
+use App\Models\Tag;
+use Illuminate\Foundation\Testing\RefreshDatabase;
+use Tests\TestCase;
+
+class TagTest extends TestCase
+{
+    use RefreshDatabase;
+
+    public function testStoreTagCreatesTag(): void
```

Comment:
Here we have test with happy path - how about unhappy path? I mean what will happen when user create tag with too long name? Or create tag that already exists? And so on